tests/test_plotter.py

```diff
@@ -450,4 +458,4 @@ def test_plot_path_3(multishells):
     pl.add_field(field_m, mesh_m)
     pl.add_field(field, mesh, style="wireframe", show_edges=True,
                  color="w", opacity=0.3)
-    pl.show_figure()
\ No newline at end of file
+    pl.show_figure()
```

Comment:
@ayush-kumar-423 could you please add tests for your new capabilities?

Reply:
@cbellot000 will do, thanks!